tidalapi/request.py

```diff
@@ -60,11 +60,25 @@ def basic_request(self, method, path, params=None, data=None, headers=None):
         request = self.session.request_session.request(method, url, params=request_params, data=data, headers=headers)
 
         refresh_token = self.session.refresh_token
-        if not request.ok and request.json()['userMessage'].startswith("The token has expired.") and refresh_token:
-            log.debug("The access token has expired, trying to refresh it.")
-            refreshed = self.session.token_refresh(refresh_token)
-            if refreshed:
-                request = self.basic_request(method, url, params, data, headers)
+        if not request.ok:
+            json_resp = None
+            try:
+                json_resp = request.json()
+            except json.JSONDecodeError:
```

Comment:
```suggestion
            except requests.JSONDecodeError:
```
See https://github.com/psf/requests/pull/5856